modules/git/command.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2015 The Gogs Authors. All rights reserved.
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2020 The Gitea Authors. All rights reserved.
```
:O

Reply:
I think it should actually be 2017 - the header was just missed.

Reply:
yup, that's exactly it.

Reply:
Actually it's 2016:

https://github.com/go-gitea/gitea/pull/450